src/components/compoundComponents/Breadcrumbs/breadcrumbs.mdx

```diff
@@ -0,0 +1,117 @@
+import BreadcrumbsExample from './BreadcrumbsExample';
+
+<CH.Scrollycoding>
+
+    # Building a compound component
+
+    <CH.Preview frameless>
+
+        <BreadcrumbsExample />
+
+    </CH.Preview>
+
+    ```tsx BreadcrumbsExample.tsx
+
+        import React from 'react';
+
+        import Breadcrumbs from './Breadcrumbs';
+
+        function App() {
+            // TODO: Replace this:
+            return (
+                <nav aria-label="Breadcrumbs">
+                    <ol>
+                        <li>
+                            <a href="/">Electronics</a>
+                        </li>
+                        <li>
+                            <a href="/cellphones">Cell Phones</a>
+                        </li>
+                        <li>
+                            <a href="/cellphones/cases" aria-current="page">
+                                Cases
+                            </a>
+                        </li>
+                    </ol>
+                </nav>
+            );
+        }
+
+        export default App;
+
+    ```
+
+    E-commerce websites often have “breadcrumbs” to help the user navigate to parent/grand-parent pages. For example:
+
+    `Electronics / Cell Phones / Cases`
+
+    Let's imagine we have the following code for these breadcrumbs (see BreadcrumbsExample.tsx).
+
+    How we can built now a compound component?
+
+    Looking at the markup, it seems like we'll need two components:
+
+    * A Breadcrumbs parent wrapper to contain the group
+
+    * A Crumb component for each link in the group
+
+    ---
+
+    Copy the chunk for the parent wrapper (Breadcrumbs) and made it generic.
+
+    ```tsx Breadcrumbs.tsx focus=4:10
+    // from ./Breadcrumbs.tsx
+    ```
+
+    ---
+
+    Copy the chunk for each link in the group (Crumb) and made it generic.
+
+    ```tsx Breadcrumbs.tsx focus=12:23
```

Comment:
The focus is not on the whole component here. on purpose?

Reply:
No. The code changed so I need to change focus.